templates/agent/prisma/dev.schema.prisma

```diff
@@ -0,0 +1,18 @@
+// This is your Prisma schema file,
```

Comment:
See [above](https://github.com/RubricLab/create-rubric-app/pull/21/files#r1403766540); let's move this to **prisma/dev/schema.prisma**